packages/api/src/bsky-agent.ts

```diff
@@ -17,7 +17,7 @@ import { sanitizeMutedWordValue } from './util'
 
 const FEED_VIEW_PREF_DEFAULTS = {
```

Comment:
Optional: could be a good time to export these and integrate them into the app instead of the duplicative code we have rn

Reply:
Don't have strong preference on this and don't want to think about it yet so I'll pass